### ISSUE TITLE
Unix.Unix_error \"Bad address\" read \"\" in asd errors

### ISSUE THREAD

**Issue by dejonghb** [openvstorage/alba#476]
```
  errors =
  [(2016/12/05_14:52:53.9438, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.6045, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.5568, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.5479, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.5469, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.4383, "(Invalid_argument Lwt_bytes.read)");
    (2016/12/05_14:52:53.4374, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.3820, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.2592, "(Unix.Unix_error \"Bad address\" read \"\")");
    (2016/12/05_14:52:53.2379, "(Unix.Unix_error \"Bad address\" read \"\")")
```

According to @domsj this is unexpected/fishy

**Comment by wimpers** [comment]
What are the consequences of these errors?

**Comment by toolslive** [comment]
It will most probably cause disqualification of the ASD.

**Comment by jeroenmaelbrancke** [comment]
I see this error also on the OVH setup:

```
 {
      "id": 52,
      "alba_id": "5364be0a-d4a0-4e0b-bc65-1079f06ea122",
      "kind": "AsdV1",
      "ips": [ "172.20.20.19" ],
      "port": 8601,
      "use_tls": false,
      "use_rdma": false,
      "albamgr_cfg": null,
      "prefix": null,
      "preset": null,
      "endpoints": null,
      "decommissioned": false,
      "node_id": "7jAgb6riiIVs1ThVLSaioszmnV374boq",
      "long_id": "DzWiVvdjcrkfWVI0aQ3MZNHMjKkbN5Sc",
      "total": 239933292544,
      "used": 214546158368,
      "seen": [
        1489489371.946834, 1489489371.945599, 1489489361.946087,
        1489489361.944648, 1489489361.944619, 1489489351.946063,
        1489489351.945792, 1489489351.944047, 1489489341.945258,
        1489489341.945183
      ],
      "read": [
        1489489300.973915, 1489489139.595749, 1489489129.961068,
        1489488672.83596, 1489488671.281515, 1489488657.754011,
        1489488190.470207, 1489487820.050904, 1489487354.834806,
        1489487351.055192
      ],
      "write": [
        1489489200.79743, 1489489184.870052, 1489488733.294509,
        1489488717.342993, 1489488252.411549, 1489488244.680496,
        1489487773.435096, 1489487769.935531, 1489487762.651841,
        1489487285.210854
      ],
      "errors": [
        [ 1489084891.297912, "End_of_file" ],
        [ 1489053040.226928, "End_of_file" ],
        [ 1489035578.421806, "End_of_file" ],
        [ 1489001640.425363, "(Unix.Unix_error \"Bad address\" read \"\")" ],
        [ 1488898764.860119, "End_of_file" ],
        [ 1488898762.146485, "End_of_file" ],
        [ 1488898707.961229, "End_of_file" ],
        [ 1488898702.280986, "End_of_file" ],
        [ 1488898697.080196, "End_of_file" ],
        [ 1488845621.750181, "Networking2.ConnectTimeout" ]
      ],
      "checksum_errors": 0,
      "claimed_since": 1488796416.133254
    },
```

**Comment by domsj** [comment]
Fixed by #705
